pages/u/[id].js

```diff
@@ -144,6 +144,16 @@ export default function Home() {
       ) : (
         <Magic />
```

Comment:
currently does not do anything. if we plan to not use it,  i would just comment it out till its ready to be utilized

Reply:
@jahmad7